acrobat/scripts/imageReplacer.js

```diff
@@ -0,0 +1,25 @@
+import { createTag } from './miloUtils.js';
+
+export default async function replacePlaceholdersWithImages(elements) {
+  createTag.then((tag) => {
+    elements.forEach((p) => {
+      const elementId = p.getAttribute('id');
+      const elementDOM = document.getElementById(elementId);
+      const cardName = elementDOM.innerHTML.toLowerCase();
+      const imageName = `credit-cards-${cardName}.jpg`;
```

Comment:
This will fail for two images because they are .png ( acrobat/img/icons/credit-cards-amex_v_mc_jcb_jpbank.png, acrobat/img/icons/credit-cards-amex_visa_mc_jcb_jpbank_jpstore.png).

Because the image names are known, maybe you can create some kind of array (or object) containing png images and use it to check whether an image is the .png or .jpg.